apps/docs/docs/core/core.md

```diff
@@ -207,3 +224,40 @@ export const contract = c.router({
   }
 });
 ```
+
+### Path Prefix
+
+The `pathPrefix` option allows you to add a prefix to paths, allowing more modular and reusable routing logic. This option is applied recursively, allowing the application of prefixes to nested contracts. In addition, when hovering over the contract, the prefixed path will appear at the beginning of the path for ease of use.
+
+Here is an example of how to use the `pathPrefix` option. In this example, the resulting path is `/api/v1/MyPath`.
+
+```typescript
+const c = initContract();
+export const contract = c.router({
+    getPost: {
+      path: 'MyPath',
```

Comment:
Make sure `path` is lowercase and a valid path to avoid confusing folks e.g `/post/:id`

what you've given here will actually be `/api/v1MyPath` at runtime